.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: CESM forums
+    url: https://bb.cgd.ucar.edu/cesm/forums/ctsm-clm-mosart-rtm.134/
```

I think this URL should be:

```suggestion
    url: https://bb.cgd.ucar.edu/cesm/forums/cam.133/
```

---

that it should! changed!